src/components/Member/Member.vue

```diff
@@ -15,7 +15,7 @@
 			:disable-menu="true"
 			:disable-tooltip="true"
 			:show-user-status="false"
-			:size="44" />
+			:size="34" />
```

Comment:
Should this also use the css variable value or is this expected to be also smaller on older versions?

Reply:
Good catch, changed now.